src/ui/page/error/i18n/error_fr.json

```diff
@@ -0,0 +1,11 @@
+{
+  "error": {
+    "renderingError": {
+      "apology": "Désolé,",
+      "message": "Une erreur s'est produite lors du traitement de votre demande.",
+      "suggestion": "Nous sommes maintenant informés de ce problème et nous nous efforçons de le résoudre. En attendant, voici ce que vous pouvez faire :",
+      "refreshButton": "Retourner à l'accueil",
+      "contactButton": "Nous contacter"
```

Comment:
I think that html tags don't have to appear in translation keys
`error.contactUs` or `error.refreshPage` should be sufficient.

Reply:
Yess, done!